packages/data-context/src/DataContext.ts

```diff
@@ -181,7 +181,7 @@ export class DataContext {
   @cached
   get cloud () {
     return new CloudDataSource({
-      fetch: (...args) => this.util.fetch(...args),
+      fetch: (...args: [RequestInfo | URL, (RequestInit | undefined)?]) => this.util.fetch(...args),
```

Comment:
ran into typings issues, my guess with the updated lock, which is kinda weird but I just chose to strongly type the lines that were failing